samples/cds-sample-application/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,34 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import nock = require('nock');
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+import cds = require('@sap/cds');
+
+describe('AppController (e2e)', () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
```

Comment:
call the method `setTestDestination()` and set the {"myTestDestination",url...} destination. Then the sample is very close to production.